.github/workflows/docker-image.yml

```
@@ -0,0 +1,40 @@
+name: Spark Iceberg image
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Docker tag to apply to this image'
+        required: true
+        type: string
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: tabulario
+          password: ${{ secrets.DOCKERHUB_ACCESS_TOKEN }}
+
+      - name: BuildX and Push
```

nit: BuildX -> Build?

---

BuildX refers to the cross-build: https://docs.docker.com/engine/reference/commandline/buildx_build/
```suggestion
      - name: Build and Push
```